android/src/main/java/com/bitwala/safetynet/RNSafetyNetModule.java

```diff
@@ -79,7 +81,14 @@ public void onSuccess(SafetyNetApi.AttestationResponse response) {
     .addOnFailureListener(activity, new OnFailureListener() {
       @Override
       public void onFailure(@NonNull Exception e) {
-        promise.reject(e);
+        if (e instanceof ApiException) {
+            // An error with the Google Play services API contains some additional details.
+            ApiException apiException = (ApiException) e;
+            promise.reject(CommonStatusCodes.getStatusCodeString(apiException.getStatusCode()), apiException.getStatusMessage());
```

Comment:
`CommonStatusCodes.getStatusCodeString(apiException.getStatusCode())` should return an error string with the error coming from the google API, eg: `NETWORK_ERROR`.